### PR TITLE
Fix ie7-8 bug that caused js errors on that.element.detach().attr('type'...

### PR DESCRIPTION
**Pull request by davidherse**
This allows the plugin to work in ie7-8

Fix ie7-8 bug that caused js errors on that.element.detach().attr('type', 'hidden'). 
Change creation of tag spans to make ie happy. 

Removed white spaces in js script. (My editor did this for me)

### CODE REVIEW

**Comment by fdeschenes** [comment]
Thank you for fixing this! :smile:
